docs/api-reference/state/observer.md

```diff
@@ -73,4 +73,45 @@ numCoins:set(25) -- prints 'coins is now: 25'
 
 -- always clean up your connections!
 disconnect()
+```
+
+-----
+
+<p class="fusiondoc-api-pills">
+	<span class="fusiondoc-api-pill-since">since v0.3</span>
+</p>
+
+### :octicons-code-24: Observer:onBind()
+
+Works the same as `:onChange()`, however it run's the specified callback
+immediately.
+
+This modified method is better to be used when using the same function to initialise an
+observer and to watch changes for it.
```

Comment:
I'm not really a fan of how these two sentences are worded. The definition relies on the other documentation for `onChange` to communicate what it does, which could lead to problems if we change the other documentation down the line without remembering to update this. It's better for documentation to stand on its own unless directly referencing another thing out of necessity.

Here's how I might write it, mirroring the structure of `onChange`:

> Connects the given callback as a change handler, and returns a function which will disconnect the callback. The callback is run immediately, and re-run whenever the observed dependency is updated.

I don't think the second sentence specifying usage patterns is a good fit for API docs. We should be describing what the API does primarily here.

Reply:
I did have a bit of trouble wording the onBind documentation haha, I'll use your one as its heaps better!